vss-extension.json

```diff
@@ -188,6 +188,17 @@
                                     "dataType": "string"
                                 }
                             },
+                            {
+                                "id": "sessionToken",
+                                "name": "Session Token",
+                                "description": "The AWS session token for signing programmatic requests.\nExample: AQoDYXdzEJr...<remainder of security token>",
```

Comment:
nit: an example is not really needed

Reply:
Removed example and added a note that this is to be used only if using external rotation mechanism